gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/tpgm/aggregation/AbstractTimeAggregateFunction.java

```diff
@@ -83,11 +97,25 @@ public PropertyValue getIncrement(TemporalElement element) {
     case TO:
       return PropertyValue.create(timeInterval.f1);
     default:
-      throw new IllegalArgumentException("Field " + field + " is not supported for time intervals" +
-        ".");
+      throw new IllegalArgumentException("Field " + field +
+        " is not supported for time intervals.");
     }
   }
 
+  /**
+   * Checks if the given property value is a temporal default value (see {@link TemporalElement}).
+   * If the temporal attribute is a {@link TemporalAttribute#TRANSACTION_TIME}, this function
+   * will return {@code false} since the transaction time is system maintained and there are no
+   * defaults to check.
+   *
+   * @param value the property value to check
+   * @return true, if the value equals a DEFAULT temporal value.
```

Comment:
The default in all caps looks odd, I think it should be replaced by either a link to the default value, `DEFAULT` in a code block or just `default`.